package.json

```diff
@@ -65,5 +65,12 @@
     "prettier": "^2.8.8",
     "turbo": "^1.9.6",
     "typescript": "^5.0.4"
+  },
+  "resolutions": {
+    "@typescript-eslint/eslint-plugin": "6.0.0-alpha.158",
+    "@typescript-eslint/parser": "6.0.0-alpha.158",
+    "@typescript-eslint/scope-manager": "6.0.0-alpha.158",
+    "@typescript-eslint/types": "6.0.0-alpha.158",
+    "@typescript-eslint/visitor-keys": "6.0.0-alpha.158"
```

Comment:
Re versioning: I'm guessing you won't want to merge this PR until we're out of alpha/beta/rc and in `6.0.0` stable.

Reply:
👍🏼 

why are the resolutions needed though?